docs/recording/recording.md

```diff
@@ -61,14 +45,28 @@ Two terminals are needed for this method: One for starting a LTTng session, anot
      ros2 run caret_demos end_to_end_sample
      ```
 
-3. Stop the target application
+2. Open another terminal and start a LTTng session with the following commands
+
+   - (Optional) `ROS_TRACE_DIR` variable is a destination directory where recorded trace data will be stored. Default is `~/.ros/tracing`
+   - With "`-s`" option, you can give session name. The recorded trace data will be stored into `~/ros_ws/evaluate/e2e_sample` in this sample
+   - Press "Enter" key to start a session
+
+   ```sh
+   source /opt/ros/humble/setup.bash
+   source ~/ros2_caret_ws/install/local_setup.bash
 
-4. Press "Enter" key to stop the LTTng session in the terminal where the LTTng session runs
+   # (Optional) Set a destination directory
+   mkdir -p ~/ros2_ws/evaluate
+   export ROS_TRACE_DIR=~/ros2_ws/evaluate
 
-<prettier-ignore-start>
-!!!info
-      A LTTng session needs to be started before running a target application. Otherwise, some trace points won't be recorded and analysis will fail later.
-<prettier-ignore-end>
+   ros2 caret record -s e2e_sample
+
+   # Start session with pressing Enter key
+   ```
+
+3. Press "Enter" key to stop the LTTng session in the terminal where the LTTng session runs
+
+4. Stop the target application
 
```

Comment:
It will be nice to add note or comment to tell that step 1 and step 2 can be reversed so that a user can record from the beginning of the target application start. It will be useful for some cases.

Also, if you add such note, please notice the user not to start a LTTng session while launching a target application especially when recording a huge application with many nodes. Otherwise, some nodes may not be recorded.

Reply:
addressed [13f6341](https://github.com/tier4/CARET_doc/pull/118/commits/13f63419058e3a7ffc2da8ae2ff58ebbbc1b7c63)

Reply:
LGTM